### ISSUE TITLE
Error generating previews when there are parentheses (1) in the name of the image filename

### ISSUE THREAD

**Issue by jay-tuckey** [nextcloud/previewgenerator#181]
Hi, love the project, it's very handy. However, I'm seeing a crash when there is a `(1)` in the name of the file that is getting a preview created for it. Here's my crash log:

```
www-data@a16560cafffc:~/html$ php occ preview:pre-generate -vvv
2020-02-03T06:38:07+00:00 Generating previews for /jtuckey/files/Photos/Google Photos/2018 Tassie Holiday Pt.2 - Before Bruny Is./MVIMG_20180430_134224.jpg
2020-02-03T06:38:07+00:00 Generating previews for /jtuckey/files/Photos/Google Photos/2018 Tassie Holiday Pt.2 - Before Bruny Is./MVIMG_20180430_134227(1).jpg
An unhandled exception has been thrown:
Error: Call to a member function file_get_contents() on null in /var/www/html/lib/private/Files/Filesystem.php:712
Stack trace:
#0 /var/www/html/lib/private/legacy/image.php(621): OC\Files\Filesystem::file_get_contents('/var/www/html/d...')
#1 /var/www/html/lib/private/Preview/Image.php(52): OC_Image->loadFromFile('/var/www/html/d...')
#2 /var/www/html/lib/private/Preview/GeneratorHelper.php(62): OC\Preview\Image->getThumbnail(Object(OC\Files\Node\File), 4096, 4096)
#3 /var/www/html/lib/private/Preview/Generator.php(205): OC\Preview\GeneratorHelper->getThumbnail(Object(OC\Preview\JPEG), Object(OC\Files\Node\File), 4096, 4096)
#4 /var/www/html/lib/private/Preview/Generator.php(127): OC\Preview\Generator->getMaxPreview(Object(OC\Files\SimpleFS\SimpleFolder), Object(OC\Files\Node\File), 'image/jpeg', '')
#5 /var/www/html/lib/private/PreviewManager.php(185): OC\Preview\Generator->getPreview(Object(OC\Files\Node\File), 32, 32, true, 'fill', 'image/jpeg')
#6 /var/www/html/custom_apps/previewgenerator/lib/Command/PreGenerate.php(205): OC\PreviewManager->getPreview(Object(OC\Files\Node\File), 32, 32, true)
#7 /var/www/html/custom_apps/previewgenerator/lib/Command/PreGenerate.php(193): OCA\PreviewGenerator\Command\PreGenerate->processFile(Object(OC\Files\Node\File))
#8 /var/www/html/custom_apps/previewgenerator/lib/Command/PreGenerate.php(164): OCA\PreviewGenerator\Command\PreGenerate->processRow(Array)       #9 /var/www/html/custom_apps/previewgenerator/lib/Command/PreGenerate.php(132): OCA\PreviewGenerator\Command\PreGenerate->startProcessing()       #10 /var/www/html/3rdparty/symfony/console/Command/Command.php(255): OCA\PreviewGenerator\Command\PreGenerate->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 /var/www/html/3rdparty/symfony/console/Application.php(915): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 /var/www/html/3rdparty/symfony/console/Application.php(272): Symfony\Component\Console\Application->doRunCommand(Object(OCA\PreviewGenerator\Command\PreGenerate), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#13 /var/www/html/3rdparty/symfony/console/Application.php(148): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#14 /var/www/html/lib/private/Console/Application.php(214): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#15 /var/www/html/console.php(99): OC\Console\Application->run()
#16 /var/www/html/occ(11): require_once('/var/www/html/c...')
#17 {main}www-data@a16560cafffc:~/html$
www-data@a16560cafffc:~/html$
```

I'm using nextcloud in a docker container. Let me know if you need any further information

**Comment by borekon** [comment]
Same error happens to me,

**Comment by joshtrichards** [comment]
Fixed in #191
